styles.css

```diff
@@ -1,3 +1,12 @@
-.body {
-  width: 100vw;
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+#books {
+  max-height: 60vh;
+  max-width: 60vw;
+  overflow-y: scroll;
```

Comment:
- [ ] The design of this section is kind of not in line with the provided wireframe. Look at this screenshot from the project requirements list. 

| The Requirement |
| --- |
| ![theirs](https://user-images.githubusercontent.com/45002654/129247633-824cd5db-255d-48e2-a571-e60b0a0eeb33.jpg) |

| The Implementation |
| --- |
| ![yours](https://user-images.githubusercontent.com/45002654/129247510-e4eba90d-0409-44aa-95e9-7995169e2b8b.jpg) |

Please consider re-writing styles for this section to make them match the wireframe.

Reply:
Of course, you are totally right, Now we fixed the styles to make them match with the wireframe :+1: 

Thank you :smile: